src/swoole/Swoole/Server.php

```diff
@@ -69,7 +69,7 @@ class Server
      * @param int $mode [optional] = \SWOOLE_PROCESS
      * @param int $sockType [optional] = \SWOOLE_SOCK_TCP
      */
-    public function __construct(string $host, int $port = 0, int $mode = \SWOOLE_PROCESS, int $sockType = \SWOOLE_SOCK_TCP)
+    public function __construct(string $host, int $port = 0, int $mode = \swoole_process, int $sockType = \SWOOLE_SOCK_TCP)
```

Comment:
This might be wrong since a constant is expected as the default mode, correct?

Reply:
PHP CS Fixer convert it to lower case automatically, I didn't notice.
I will fix and push it again.